webui/src/store/modules/user.js

```diff
@@ -155,8 +155,11 @@ export default {
                                 // Optimization: convert to base64 here and not in the OrgTree component
                                 // to have ready-to-use values cached in the user object
                                 if(data) {
-                                    let base64 = btoa(String.fromCharCode.apply(null, data));
-                                    elem.photo = 'data:image/jpeg;base64,' + base64;
+                                    let reader = new FileReader();
+                                    reader.readAsDataURL(data);
+                                    reader.onloadend = function() {
```

Comment:
arrow function may be?
is it Ok to provide callback after you already started the operation? (let's change onloaded and readas lines?)

Reply:
Unfortunately it does not work on Windows :( So I have to support both Blobs and typed arrays. Will fix now.